### DISCUSSION TITLE
Звуки

### DISCUSSION THREAD

**Discussion by nasar75** [OGSR/OGSR-Engine#286] [Ideas]
У оружия есть параметр 
anim_reload_partly  = 

А вот звук
snd_reload_partly =
не поддерживается. Проигрывается snd_reload .

Непорядок...

**Comment by xrSimpAI** [comment]
Ну как бы неполная перезарядка - это как правило та же анимация перезарядки, просто обрезанная по времени.
Звук по окончании анимаций перезарядки принудительно останавливается, поэтому не делали отдельный.

**Comment by nasar75** [comment]
Так-то да, вот токмо у неполной перезарядки патроны в магазине сразу есть, а у полной перезарядки магазин сначала пустой...

Зы. Это я к чему написал. Перенес СВД из ганслигера на ТЧ, правда, лишние кости поудалял (токмо на стволе было 46 штук), кость прицела оставил одну.
Так у этой свд есть анимки клина и затворной задержки.
Ежели нада, залью геймдату.

В планах перенести сву - у нее еще и анимация переключения режимов огня, и АПС - там вдобавок куча анимок неполной перезарядки с разным количеством патронов в магазине.
Делать?...

**Comment by xrSimpAI** [comment]
Ну, snd_reload_partly я могу наверно сделать в будущем, если очень надо.
А всё остальное - нет.
Вообще жаль что ганслингер базируется не на нормальном движке с исходниками, а на ассемблерных врезках в ориг. двиг. А то можно было бы перенести его весь в огср без особых проблем.

**Comment by rutrash** [comment]
Можно ссылочку на СВД?

**Comment by molodoy-chelovek** [comment]
@KRodinn Приветствую! 
Извиняюсь за оффтоп, но этот вопрос не стоит создания отдельного топика.
Ванильный ТЧ не поддерживает 16 bit motions. Поддерживает ли их OGSR?

**Comment by xrSimpAI** [comment]
Конечно. OGSR поддерживает модели из SDK 0.7 же.

**Comment by nasar75** [comment]
Жалко, конечно, ну да ладно... Меньше работы...
СВД тут: https://www.amk-team.ru/forum/topic/8216-the-new-weapon-and-its-addition-in-stalker-soc/?do=findComment&comment=1358242

А еще вопрос.
В гансе и адаптации на ЗП анимированные аптечки\водки используют слот артефакта (десятый вроде).
Сложно ли добавить этот слот в OGSR?

**Comment by xrSimpAI** [comment]
Тут это и через восьмой слот нормально делается, мне вот видос скидывали https://yadi.sk/i/V3q8uqDsm9z7Tw

**Comment by seaz5150** [comment]
> Тут это и через восьмой слот нормально делается, мне вот видос скидывали https://yadi.sk/i/V3q8uqDsm9z7Tw

Could you share a link to these item animations?

**Comment by xrSimpAI** [comment]
No.

**Comment by seaz5150** [comment]
I cry. Anyway good job making them work)

**Comment by xrLil-Batya** [comment]
> Вообще жаль что ганслингер базируется не на нормальном движке с исходниками, а на ассемблерных врезках в ориг. двиг. А то можно было бы перенести его весь в огср без особых проблем.
https://github.com/gunslingermod/GunsXRay/tree/gunslinger_wpnpatch

**Comment by xrSimpAI** [comment]
Это я видел, там же недодел какой-то, прям совсем сырой.
